src/base/password_log.cpp

```diff
@@ -0,0 +1,895 @@
+/*
+ * Copyright 2008 Search Solution Corporation
```

Comment:
Only the cubrid copyright is described for the newly created file.

Reply:
Okay. 
Thanks.

Reply:
@kisoo-han 
If I do as you said, the license check in CI will fail.
So, we'll stick with the original wording again.